.github/workflows/rust-tests.yml

```diff
@@ -11,8 +11,8 @@ on:
 env:
   CARGO_TERM_COLOR: always
   CARGO_REGISTRIES_CRATES_IO_PROTOCOL: sparse
-  RUST_VERSION_OLD: "1.68.0"
-  RUST_VERSION_NEW: "1.88.0"
+  RUST_VERSION_OLD: "1.82" # aka MSRV
```

Comment:
Why do you need rust newer, than 1.68?

It's ok to update an old version, when necessary, but the update should a reasonable justification.

Reply:
Because some of updated dependencies now require Rust `1.82`
Probably I can downgrade some dependencies to keep MSRV `1.81` but not less.

Updates of dependencies which contains fixes for critical vulnerabilities reported by `cargo audit` requires at least `1.81`.